_posts/newsletter/2020-04-09-this-month-in-solid.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: newsletter
+title: This Month in Solid 2020-04-09
```

Comment:
Update date?

Reply:
It was a bit delayed due to having to make sure all links kept working correctly with the switch to a monthly newsletter, and Inrupt having an All-hands, but the content covers the period up to that date, so we're backdating it indeed.